src/components/character-avatar/styles.ts

```diff
@@ -40,11 +40,64 @@ export const styles = {
       "& > svg > g": {
         "& > path": {
           fill: fill,
+          stroke: fill,
         },
         "& > g > path": {
           stroke: fill,
         },
       },
     };
   },
+  animation: (stage: MatchStage, isSelected?: boolean, isBot?: boolean) => {
+    const determineBorderFill = () => {
+      if (isSelected) {
+        if (isBot) {
+          return `url("data:image/svg+xml,%3Csvg width='100%' height='100%' viewBox='0 0 264 284' fill='none' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='M3 249.872L3 280.342L33 280.342' stroke='%235CFF00' stroke-width='4'/%3E%3Cpath d='M261 249.872L261 280.342L231 280.342' stroke='%235CFF00' stroke-width='4'/%3E%3Cpath d='M3 33.5952L3 3.125L33 3.125' stroke='%235CFF00' stroke-width='4'/%3E%3Cpath d='M261 33.5952L261 3.125L231 3.125' stroke='%235CFF00' stroke-width='4'/%3E%3C/svg%3E") 4`;
```

Comment:
Add some comments to explain what's the SVG code rendering. Or put it in an independent asset file "portrait-select-border".